dietpi/dietpi-software

```diff
@@ -1591,6 +1591,13 @@ Available commands:
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/distributed_projects/#yacy'
 		aSOFTWARE_DEPS[$software_id]='196'
 		#------------------
+		software_id=141
+		aSOFTWARE_NAME[$software_id]='adsb-feeder'
```

Comment:
What is your favourite beautified name for this? "ADSB-Feeder", "ADSB Feeder" or "ADS-B Feeder"? 😄
I like to have it consistent everywhere in our scripts and docs and aligned with the preferred spelling of upstream dev(s).

Reply:
ADS-B Feeder